### ISSUE TITLE
backtick open not shown in eslint --init

### ISSUE THREAD

**Issue by nhooyr** [eslint/eslint#7997]
**What version are you using?**
v3.14.1

**What did you do?**
ran `eslint --init`

**What happened?**
for quotes, it only showed single or double

**What did you expect to happen?**
it should give the options of single, double or backtick.

**Comment by platinumazure** [comment]
Hi @nhooyr, thanks for the issue.

Could you please be more explicit in your repro steps? I assume you're referring to the "answer questions about your desired configuration" option but it's not clear.

**Comment by nhooyr** [comment]
Yes, "answer questions about desired configuration" option, and then "What quotes do you use for strings?" shows just Single or Double.

**Comment by ilyavolodin** [comment]
`init` was added before `quotes` rule had support for backticks. I guess we can update `init` step to include them now. Although, if I remember correctly, there were some edge-cases with using backticks everywhere, so I'm not 100% sure this is a good idea.

**Comment by alberto** [comment]
I think this is a valid request. If the rule can deal with backticks, I don't see a reason to not include it as an option.

@ilyavolodin i think the edge-cases where ironed out. and if they aren't, then it's a bug in the rule that we should deal with.

I'm championing this.

**Comment by alberto** [comment]
@eslint/eslint-team opinions on this? there is already a PR implementing it.

**Comment by platinumazure** [comment]
:+1: from me. I left this hanging after waiting for OP to clarify the repro steps, oops.

**Comment by michaelficarra** [comment]
I think the backtick option should be discouraged. :-1:

**Comment by alberto** [comment]
@michaelficarra can you explain why?

**Comment by michaelficarra** [comment]
Because this rule doesn't just enforce quoting of strings. It also enforces quoting of property names, module specifiers, etc. where backticks are not allowed. So, in my opinion, it's a broken option based on a bad premise.

**Comment by alberto** [comment]
Can you provide an example where the rule enforces invalid code?

This is correct code for `"quotes": ["error", "backtick"]`:
```js
import foo from 'bar';

var a = {
  "b": `c`
};
```

**Comment by mikesherov** [comment]
I think the point @michaelficarra is making is that folks think of `'` and `"` as interchangeable, which they are. And then they think backtick is interchangeable and it isn't. 

And you're inevitably left with needing to choose 2 quote styles, backticks usually, and either single or double when you can't.

Better would be an option to enforce single/double, with optionally enforcing backticks for concatenation.

**Comment by mikesherov** [comment]
And @michaelficarra isn't making that point, then I am ;-)

**Comment by alberto** [comment]
Thanks for your interest in improving eslint. Unfortunately, it looks like this issue didn't get enough support from the team and so I'm closing it. While we wish we'd be able to accommodate everyone's requests, we do need to prioritize. We've found that issues failing to reach consensus after 21 days tend to never do it, and as such, we close those issues. This doesn't mean the idea isn't interesting, just that it's not something the team can commit to.
